bootstrap/pkg/kfapp/gcp/gcp.go

```diff
@@ -765,15 +743,7 @@ func (gcp *Gcp) updateDM(resources kftypes.ResourceEnum) error {
 		}
 	}
 
-	// Setup kube config
-	// TODO(#3061): figure out how to properly build config without kubeconfig.
-	// TODO(jlewi): Not sure about #3061. I think we only want to run gcloud get-credentials if running
-	// as a CLI and not when running on click to deploy. Should we check TokenSource? What's the best
-	// way to tell whether we are running in cluster?
-
-	accessToken, _ := gcp.kfDef.GetSecret(GcpAccessTokenName)
```

Comment:
Are we still using this token secret?

Reply:
Not anymore. This was originally used as a way for click to deploy to inject the OAuth credential.
But we changed the code so Gcp.go now has a token source and code just calls that to get the token.